pkg/trojan/export_test.go

```diff
@@ -0,0 +1,8 @@
+package trojan
```

Comment:
is this a new pattern we use in bee? interesting

Reply:
i think it is a general golang practice...But i saw this first in bee code like you